xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -997,4 +1000,36 @@ void CGUIFontTTFBase::EmboldenGlyph(FT_GlyphSlot slot)
   slot->metrics.vertAdvance  += dy;
 }
 
+// Lighten code - original taken from freetype2 (ftsynth.c)  
+void CGUIFontTTFBase::LightenGlyph(FT_GlyphSlot slot)
+{
+  if (slot->format != FT_GLYPH_FORMAT_OUTLINE)
+    return;
+
+  /* some reasonable strength */
+  FT_Pos strength = FT_MulFix(m_face->units_per_EM,
+                              m_face->size->metrics.y_scale) / -48;
```

Comment:
Unfortunately @phil65 I have zero C++ knowledge and this is mostly me working out how bold works and re-engineering it for light, plus a lot of help from @mkortstiege and @Paxxi.